dune-project

```
@@ -12,6 +12,17 @@
 (source (github ocsigen/lwt))
 (documentation "https://ocsigen.org/lwt")
 
+(package
+ (name lwt_retry)
+ (synopsis "Utilities for retrying Lwt computations")
+ (authors "Shon Feder")
+ (maintainers
+   "Raphaël Proust <code@bnwr.net>"
+   "Shon Feder <shon.feder@gmail.com>")
+ (depends
+  (ocaml (>= 4.08))
+  lwt))
```

```suggestion
  (lwt (>= 5.3.0))))
```

because of `Lwt.Syntax`